tests/php/CoverageTest.php

```diff
@@ -77,4 +77,28 @@ public function testSubletCoverage()
         }
     }
 
+    public function testValidateJsonFormat()
+    {
+        // Move this to setUp if another tests uses this
+        $lists = json_decode(file_get_contents($this->root . '/src/IFSC.json'), true);
+
+        $failures = [];
+
+        foreach ($lists as $bankCode => $list) {
+            foreach ($list as $partialCode) {
+                if (is_string($partialCode) and (strlen(trim($partialCode)) !== 6))
+                {
+                    $failures[] = [$bankCode, $partialCode];
+                }
+            }
+        }
+
+        if(count($failures) > 0)
+        {
+            foreach ($failures as $failure) {
+                echo "{$failure[0]}: {$failure[1]}\n";
```

Comment:
The test needs to be force-failed. Perhaps a assert length 0 on the count?

Reply:
yep